### PR TITLE
Link to the "data transparency" page inside installer

### LINKED ISSUE #31625: As a merchant I want to know what is tracked during the modules installation step (installer)
As a merchant I want to be informed what data are tracked in the software during modules installation 

## Context

When prestashop is being installed, the project wishes to collect the shop (being installed) URL and the version of PrestaShop being used.
Even if this is compliant with GDPR we wish the user to be fully informed.

## Business rules

During the shop installation (inside installer) - Step 2  (license) 
Add 
- A specific Title (legal information <- Need wording ) 
- A textfied  (need wording)  with 
     - a message / text : it should warn the user that the shop URL will be collected  (need wording) 
     -  add a link to a specific webpage on https://www.prestashop-project.org/ (see https://github.com/PrestaShop/open-source/issues/166)

## Wording

Provided directly in PR by @l-delin : https://github.com/PrestaShop/PrestaShop/pull/31697#issuecomment-1466090061

### PR DESCRIPTION
**Pull request by lartist**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 8.0.x
| Description?      | Adding link to the data transparency from the project's website
| Type?             | improvement
| Category?         | IN
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | go to /install-dev/index.php and see if the message is well display (see the image  further in the description)
| Fixed ticket?     | Fixes #31625
| Related PRs       | 
| Sponsor company   |

Capture d’écran 2023-03-09 à 11 31 04

### CODE REVIEW

**Comment by lartist** [comment]
@fabienvallon, @l-delin please tell me more about the wording.
I had no graphic rules so I've used an already existing one (info block). If you want me to change anything let me know.

File: install-dev/theme/views/license.php
Comment (kpodemski) [inline install-dev/theme/views/license.php]:
```suggestion
        'Know that some of the store's public and technical data may be collected. If you want more information, see this page:', [], 'Install'
```

What if it'd be URL and PHP in the future? I think keeping the details on the project's website is better.

File: install-dev/theme/views/license.php
Comment (jolelievre) [inline install-dev/theme/views/license.php]: I guess this one is blocked until the page is ready right?

Comment (0x346e3730) [inline install-dev/theme/views/license.php]:
```suggestion
        <a href="https://www.prestashop-project.org/" target="_blank">https://www.prestashop-project.org/</a>
```

**Review by 0x346e3730 — DISMISSED** [review]
Small whitespace to remove but lgtm otherwise

File: install-dev/theme/views/license.php
Comment (l-delin) [inline install-dev/theme/views/license.php]:
```suggestion
        'Some of your store\'s public and technical data will be collected. If you want more information, see this page:',
```

**Review by l-delin — COMMENTED** [review]
edit for clarity :-)

**Comment by eternoendless** [comment]
I think the wording is too ambiguous and fear-inducing this way.

AFAIK this is only true when one specific module is installed, that you can choose not to install during the process. Wouldn't it make more sense to make the module show the message?

If we need to keep it generic to speed things up, I suggest to change this way:

> ### Privacy note
> 
> Some project modules may submit public and technical information about your shop to the PrestaShop Project for analytics purposes. To learn more and make an informed choice, read [this article](link-goes-here).

**Comment by l-delin** [comment]
> ### Privacy note
> 
> Some project modules may submit public and technical information about your **store** to the PrestaShop Project for analytics purposes. To learn more and make an informed choice, read [this article](link-goes-here).

:-)

**Review by matks — DISMISSED** [review]
Warning ⚠️ : we had to perform a little git history rewriting when 8.0.2 was released because of an issue when publishing the security changes.

I think your pull request is impacted by this :( only the commit https://github.com/PrestaShop/PrestaShop/pull/31697/commits/cebbb0438d800856e0ecd0a093b720611c3eebef should be in it but a lot more content and commits now are in your PR. You need to rebase or cherry-pick

Sorry 😞

**Comment by matks** [comment]
About the wording: OK let's settle for @l-delin choice.

```
Privacy note

Some project modules may submit public and technical information about your store to the PrestaShop Project for analytics purposes. To learn more and make an informed choice, read this article.
```

I agree with @kpodemski : the problem is that this text is hardcoded. So it cannot be changed: in PrestaShop 8.0.3 it will remain like this forever. This is why the content must be global enough to accommodate future changes. Even if we modify the text in 8.0.4 or 8.0.5, there might be people still using 8.0.3 then the text will become irrelevant in the future when telemetry changes.

On the other hand the webpage content can be updated everytime needed: this is where the accurate informations must go.

Next time @l-delin and @eternoendless it would be better to discuss the wording on the issue https://github.com/PrestaShop/PrestaShop/issues/31625 😛 not on the PR.

File: config/smartyadmin.config.inc.php
Comment (FabienPapet) [inline config/smartyadmin.config.inc.php]: This method has been removed IIRC, use `postProcessTranslation`

**Review by FabienPapet — DISMISSED** [review]

File: config/smartyadmin.config.inc.php
Comment (lartist) [reply config/smartyadmin.config.inc.php]: @FabienPapet it's not part of this PR. It's due to the force push of @matthieu-rolland. I'll cherry pick and it would be solved

**Comment by lartist** [comment]
I've made all changes required. It should be ok. Just think to merge https://github.com/PrestaShop/open-source/pull/169 ASAP because it contains a new page which is used as an href in this PR.

**Review by mflasquin — APPROVED** [review]

**Review by 0x346e3730 — APPROVED** [review]

**Review by florine2623 — APPROVED** [review]
Hello @lartist ,

LGTM
Screenshot 2023-03-15 at 12 17 09

⚠️ Waiting for https://github.com/PrestaShop/open-source/pull/169 to be merged before merging this PR, as it redirects to a 404 page at the moment.

Screenshot 2023-03-15 at 12 18 00

Thanks!

**Review by jolelievre — APPROVED** [review]

**Review by FabienPapet — APPROVED** [review]

**Comment by FabienPapet** [comment]
Thank you @lartist

**Pull request merged**
